### PR TITLE
don't close wg interface on soft reset

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

**Review by abhishek9686 — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
tested and passed

File: functions/daemon.go
Comment (abhishek9686) [inline functions/daemon.go:1]: On soft restart, you shouldn't stun, The bug was reported by @theguy951357, but I don't see any changes for that

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: functions/daemon.go
Comment (mattkasun) [reply functions/daemon.go:1]: Not reproducable

Comment (abhishek9686) [reply functions/daemon.go:1]: even if it's not reproducible, it's pretty evident it can happen again

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**